examples/tests/valid/reassignment.w

```diff
@@ -3,3 +3,25 @@ assert(x == 5);
 
 x = x + 1;
 assert(x == 6);
+
+resource R {
+  var f: num;
+  init() {
+    this.f = 1;
+  }
+  
+  inc() {
+    this.f = this.f + 1;
+  }
+}
+
+let r = new R();
+r.inc();
+assert(r.f == 2);
+
+let f = (var arg: num):num => {
+  arg = 0;
+  return arg;
+}
+
+assert(f(1) == 0);
```

Comment:
nit: It could be worth testing this case:
```suggestion

let y = 1;
assert(f(y) == 0);
assert(y == 1); // y is not modified, since Wing functions are pass-by-value

```

Reply:
Why not, i'll add it.